accepted/build/dotnet-patch-check.md

```diff
@@ -0,0 +1,27 @@
+# Getting latest .NET Core Runtime patch information
+
+## Goal
+
+When patches are released for the .NET Core runtime, developers should not need to install an entirely new .NET Core SDK.  They should be able to install only the updated runtime as needed, and the existing SDK should have a way of discovering that new patches should be used for self-contained applications.
+
+Eventually we hope to be able to stop producing new releases of the .NET Core SDK whenever there is a patch to the runtime.  This proposal is a step in that direction, however a lot more will be needed to enable that.
+
+## Proposal overview
+
+- Add a `dotnet` command which will check for the latest patches to the .NET Core runtime.  Naming is TBD, straw man for this proposal is `dotnet patch-check`.  The command will:
+    - Update the latest patch versions to use for self-contained applications
+    - Notify the user if the installed shared frameworks are not up-to-date, with a link to download the latest patch
```

Comment:
How does that work for Linux distributions? If .NET Core is a part of a Linux distributions, the update is handled by the package manager and pretty much anything a user does to install the latest version manually will be wrong or conflict with the package manager.